packages/bootleg-app-contracts/migrations/2_deploy_and_mint.js

```diff
@@ -1,17 +1,34 @@
 const BootlegToken = artifacts.require('BootlegToken');
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
 
-module.exports = async deployer => {
+module.exports = async (deployer, network, accounts) => {
+  let firstOwner = web3.utils.toHex(0);
+  const metaDataURI = 'https://ipfs.infura.io/ipfs/QmSomeHash';
+
+  // Bootleg Token ID
   const tokenId = web3.utils.toBN(1);
+  // should be the Band
+  firstOwner = accounts[0];
+  // should be the Bootlegger
+  secondOwner = accounts[1];
 
-  const accounts = await web3.eth.getAccounts();
+  // Deploy our BootlegToken
+  await deployer.deploy(BootlegToken, 'Bootleg', 'BLEG', 25);
```

Comment:
Interesting find. I had thought `deployer.deploy` returned a contract instance making the next line removable. Instead, I found that 1, still don't know about the first question, but, you can pass multiple contracts to be deployed! 

https://truffleframework.com/docs/truffle/getting-started/running-migrations#deployer-deploy-contract-args-options-

Just an interesting find, you have dependencies here that you wouldn't be able to use this feature 🤷‍♂️

Reply:
👍 Yeah, because of our circular dependency fun (Trading contract must know about token contract and token contract must know about trading contract 🤦‍♂️ ) doing both at once would be problematic... at least based on how the code is currently written.